TranslatorReasonerAPI.yaml

```diff
@@ -208,12 +214,24 @@ components:
         - id
     EdgeBinding:
       type: object
+      description: >-
+        A instance of EdgeBinding is a single KnowledgeGraph Edge mapping,
+        identified by the corresponding 'id' object key identifier of the
+        Edge within the Knowledge Graph. Instances of EdgeBinding may include
+        extra annotation (such annotation is not yet fully standardized).
       properties:
         id:
           type: string
-          description: A knowledge-graph edge id, i.e. the `id` of a KEdge
+          description: The key identifier of a specific KnowledgeGraph Edge.
+        score:
```

Comment:
I think the `score` property should be introduced and discussed in a separate PR from the documentation PRs since it's a functional change to TRAPI.

The same comment may also apply for specifying `additionalProperties: True` . This change doesn't functionally change anything since that's the default behavior anyway, but this may give the impression that an explicit decision has been made when I'm not sure one has been made. I'll defer this to @edeutsch and @patrickkwang

Reply:
Agreed, adding this score property is a functional change, and should be independent of these documentation updates. `additionalProperties: true` is not a functional change since it is the default, and I think it's a good idea to make it explicit for readability. @edeutsch made the same addition in a separate PR.

Reply:
Done.  See Commit  https://github.com/NCATSTranslator/ReasonerAPI/pull/170/commits/1c96bb08015ed286a0da1e5873a02e8d1fb9a216